lib/react/rails/railtie.rb

```diff
@@ -97,6 +97,20 @@ class Railtie < ::Rails::Railtie
         # The class isn't accessible in the configure block, so assign it here if it wasn't overridden:
         app.config.react.server_renderer ||= React::ServerRendering::BundleRenderer
 
+        def yellow(message)
+           "\e[33m#{message}\e[0m"
+        end
+
+        node_version = `echo $(node -v | cut -c 2-)`.strip
```

Comment:
We should avoid calling out to shell where possible.
ExecJS is already a dependency of ours and can abstract this cross-os for us.

See comment on checking JS runtime;
https://github.com/reactjs/react-rails/issues/947#issuecomment-431569700
Also see code on ExecJS Runtime abstraction:
https://github.com/rails/execjs/blob/master/lib/execjs/runtimes.rb

In all honesty I'm tempted to see if they'd accept a PR that would give each runtime either versions or capabilities, which we could make use of.

Reply:
Thanks for the review! I will address the comments later today and update

Reply:
How's it going?